### ISSUE TITLE
Cannot save Inline-Widget

### ISSUE THREAD

**Issue by Scentyoo** [kartik-v/yii2-datecontrol#59]
I cannot save the date when using the DatePicker::TYPE_INLINE
It happens, because **the hidden input field doesn't get updated**

If I remove the type it works as expected. Here is my config:
```
<?= $form->field($model, 'startdate')->widget(DateControl::classname(), [
    'type'=>DateControl::FORMAT_DATE,
    'options' => [
        'type' => DatePicker::TYPE_INLINE,    // if I comment out this line it works
        'pluginOptions' => [
            'todayHighlight' => true,
        ],
    ],
]); ?>
```

**Comment by kartik-v** [comment]
This is resolved with release v1.3.3. of yii2-widget-datepicker.

**Comment by romalat** [comment]
The same problem with DateTimePicker widget
